### DISCUSSION TITLE
Pywinauto Observations/Issues

### DISCUSSION THREAD

**Discussion by rupakula** [pywinauto/pywinauto#1396] [Q&A]
We are observing the following issues w.r.t Pywinauto 0.6.7.
Could you please kindly review the following issues and help us in this context.

1. 	There are some buttons in **ToolsStrip1** control.
	WinSpy,Spy++ detect ToolStrip but unable to identify the individual controls(Buttons).
	We tried print_control_identifiers(),children() and descendents() ,but we are unable to get the Button Controls.
	Can you please suggest if there is there is any Inspection tool to identify these controls.(or) any Pywincontrol to check the children.

2. 	We were trying to detect a control under **menubar** control . 
	I tried MenuBarClickInput API but it shows as the error as "Neither GUI element (wrapper) nor wrapper method 'MenuBarClickInput' were found (typo?)"
	Since this is mentioned in https://stackoverflow.com/questions/33113913/pywinauto-menuselect-cannot-be-used-to-select-menubar-in-some-applications.
	Can you please suggest if there is any other API to access this control.(This API is also mentioned in https://pywinauto.readthedocs.io/en/latest/code/pywinauto.controls.common_controls.html)
	
3. We have a Button Control with
	ClassName: WindowsForms10.BUTTON.app.0.bb8560_r8_ad1
	Instance Num:3
	Control Name:btnName
	When we use ControlName(btnName) with click_input() -It throws no error/exception but nothing happens
	When we use ControlName(btnName) with click() -It throws error as "Element Not Enabled"
	When we use Button3(ClassName+Instance) both click and click_input works .
	Can you please share your inputs as why the ControlName does not work.
	
Thanks in advance
Prathyusha R

**Comment by cassini1010** [comment]
For the issue number 1, you can try performing print_control_identifiers once with 'uia' as a backend and once with 'win32' as backend you might get the button controls.
